sonar-orchestrator/src/main/java/com/sonar/orchestrator/server/ServerInstaller.java

```diff
@@ -259,51 +228,31 @@ private Properties configureProperties(SonarDistribution distribution, Packaging
     properties.putAll(databaseClient.getAdditionalProperties());
     setIfNotPresent(properties, "sonar.log.console", "true");
     InetAddress webHost = loadWebHost(properties, loopbackHost);
-    configureSearchProperties(properties, loopbackHost, packaging);
+    configureSearchProperties(properties, loopbackHost);
     properties.setProperty(WEB_HOST_PROPERTY, webHost.getHostAddress());
     properties.setProperty(WEB_PORT_PROPERTY, Integer.toString(loadWebPort(properties, webHost)));
     setIfNotPresent(properties, WEB_CONTEXT_PROPERTY, "");
     completeJavaOptions(properties, "sonar.ce.javaAdditionalOpts");
     completeJavaOptions(properties, "sonar.search.javaAdditionalOpts");
     completeJavaOptions(properties, "sonar.web.javaAdditionalOpts");
-
-    if (!distribution.isDefaultForceAuthentication() && packaging.getVersion().isGreaterThanOrEquals(8, 6)) {
-      //disable enforcing authentication, as it has been enabled by default starting from 8.6
-      setIfNotPresent(properties, FORCE_AUTHENTICATION_PROPERTY, "false");
+    if (!distribution.isDefaultForceAuthentication()) {
+      setIfNotPresent(properties, "sonar.forceAuthentication", "false");
     }
-
-    if (!distribution.isForceDefaultAdminCredentialsRedirect() && packaging.getVersion().isGreaterThanOrEquals(8, 8)) {
-      //disable enforcing the redirect to change the default admin password, as it has been enabled by default starting from 8.8
-      setIfNotPresent(properties, FORCE_REDIRECT_DEFAULT_ADMIN_CREDENTIALS, "false");
```

Comment:
I'm a bit worried about these changes especially if they will impact the languages team 🤔 maybe we should send an email after orch release, so that engineering teams are aware

Reply:
Yeah I agree. Theoretically speaking, we could expect that no one uses the orchestrator to start an unsupported version of SQ, but in practice some might have fixed-version tests that should likely be cleaned up now. I'll send an email to dev@sonarsource.com once the release is done. 
Thanks Jacek 🙏